compose/.idea/misc.xml

```diff
@@ -13,10 +13,11 @@
       <item index="3" class="java.lang.String" itemvalue="org.openjdk.jmh.annotations.BenchmarkMode" />
     </list>
   </component>
+  <component name="ExternalStorageConfigurationManager" enabled="true" />
```

Comment:
Is this required by the bump / worth doing in this PR? 

Also – for my own education – what is this? A Google search + search in #compose were surprisingly light on details.

Reply:
Idk, intellij man. It's been a while since we've updated this, so I'd rather just commit it than continue fighting it.

Reply:
https://github.com/square/workflow-kotlin/pull/438